plugin/src/main/java/com/denizenscript/denizen/utilities/command/DenizenCommandHandler.java

```diff
@@ -279,7 +283,7 @@ public void version(CommandContext args, CommandSender sender) throws CommandExc
      */
     @Command(
             aliases = {"denizen"}, usage = "save",
-            desc = "Saves the current Denizen save data to file as needed.", modifiers = {"save"},
+            desc = "Saves the current Denizen save data to disk as needed.", modifiers = {"save"},

```

Comment:
Incorrect change

Reply:
When I made this change I was thinking that for most users, "file" and "disk" were synonymous, I will revert this.